src/index.js

```diff
@@ -4,51 +4,79 @@ import '../materials/internals/grid.mod.css';
 import '../materials/internals/reset.mod.css';
 
 /* eslint-disable max-len,prettier/prettier */
-import * as i18n from '../i18n'
+import * as i18n from '../i18n';
```

Comment:
@emmenko originally turned of prettier for this specific file, as indicated by the `/* eslint-disable max-len,prettier/prettier */` comment. This file is also part of `.pretterignore`.

The intention of turning prettier of was to keep the exports as one-liners which makes them a bit easier to read (subjective).

Seems like your editor did not respect this setting and that's where the diff is coming from.

To me the inconvenience of turning prettier on/off for this one file outweighs the improved readability. I would be okay with enabling prettier for this file, removing the `/* eslint-disable prettier/prettier */` comment and removing this file from `.prettierignore`.

WDYT @emmenko ?

Reply:
I opened #236 which removes the exclusion.